apps/ledger-live-desktop/src/newArch/features/LNSUpsell/components/LNSbannerCard/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { t } from "i18next";
+import { Trans } from "react-i18next";
+import React from "react";
+import { BannerCard, Text } from "@ledgerhq/react-ui";
+import type { FlexBoxProps } from "@ledgerhq/react-ui/components/layout/Flex/index";
+import { AnalyticsButton, AnalyticsPage } from "LLD/features/LNSUpsell/types/enum/Analytics";
+import { openURL } from "~/renderer/linking";
+import { track } from "~/renderer/analytics/segment";
+
+type Props = Omit<FlexBoxProps, "maxHeight"> & {
+  type: "optIn" | "optOut";
+  location: "manager" | "accounts";
+  image: string;
+  ctaLink: string;
+  learnMoreLink: string;
+  discount: number;
+};
+
+export function LNSBannerCard({
+  type,
+  location,
+  image,
+  ctaLink,
+  learnMoreLink,
+  discount,
+  ...boxProps
+}: Props) {
+  const handleCTAClick = () => {
+    track("button_clicked", {
+      button: AnalyticsButton.CTA,
+      link: ctaLink,
+      page: AnalyticsPageMap[location],
+    });
+    openURL(ctaLink);
+  };
+  const handleLearnMoreLink = () => {
+    track("button_clicked", {
+      button: AnalyticsButton.LearnMore,
+      link: learnMoreLink,
+      page: AnalyticsPageMap[location],
+    });
+    openURL(learnMoreLink);
+  };
+
+  return (
+    <BannerCard
+      {...boxProps}
+      title={t(`lnsUpsell.banner.${location}.${type}.title`)}
+      description={
+        <Trans i18nKey={`lnsUpsell.banner.${location}.${type}.description`}>
+          <Text color="constant.purple">{{ discount } as never}</Text>
+        </Trans>
+      }
+      image={image}
+      cta={t(`lnsUpsell.banner.${location}.${type}.cta`)}
+      linkText={t(`lnsUpsell.banner.${location}.${type}.linkText`)}
+      descriptionWidth={320}
+      maxHeight={175}
+      onClick={handleCTAClick}
+      onLinkClick={handleLearnMoreLink}
+    />
+  );
+}
+
+const AnalyticsPageMap: Record<Props["location"], AnalyticsPage> = {
+  manager: AnalyticsPage.Manager,
+  accounts: AnalyticsPage.Accounts,
+};
```

Comment:
You should move this to`apps/ledger-live-desktop/src/newArch/features/LNSUpsell/types/enum/Analytics.ts` maybe ?

Reply:
IMO the types folder should only contain types, interfaces, and enums. WDYT ?

Reply:
<img width="226" alt="Screenshot 2025-02-12 at 09 38 37" src="https://github.com/user-attachments/assets/404769f9-46cc-4a95-ad2b-eb9e73bd1a4e" />
vs
<img width="197" alt="Screenshot 2025-02-12 at 09 38 55" src="https://github.com/user-attachments/assets/26c387e3-a804-42b5-9c1e-ad4bd6a91b48" />

I think borderRadius is missing?